druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/ExtractionNamespace.java

```diff
@@ -10,7 +10,6 @@
 @JsonSubTypes(value = {
         @JsonSubTypes.Type(name = "mahajdbcleaderfollower", value = JDBCExtractionNamespaceWithLeaderAndFollower.class),
         @JsonSubTypes.Type(name = "mahajdbc", value = JDBCExtractionNamespace.class),
-        @JsonSubTypes.Type(name = "maharocksdb", value = RocksDBExtractionNamespace.class),
```

Comment:
Not backwards compatible.  This was added for a reason, see #418

Reply:
This line and the one after it don't agree with each other.  The registration in the class above (JsonTypeName) declares one name, while here we are using two (one of which is undeclared) so whenever an ExtractionNamespace calls whichever of the two names isn't mentioned in RocksDBExtractionNamespace.java the json type will come up as undeclared.  I changed it back to mahainmemorydb to keep it consistent with the current usage.